diff --git a/node_modules/@digdir/designsystemet-react/dist/types/components/Button/Button.d.ts b/node_modules/@digdir/designsystemet-react/dist/types/components/Button/Button.d.ts
index 8046351..e1cb98d 100644
--- a/node_modules/@digdir/designsystemet-react/dist/types/components/Button/Button.d.ts
+++ b/node_modules/@digdir/designsystemet-react/dist/types/components/Button/Button.d.ts
@@ -89,7 +89,7 @@ export declare const Button: React.ForwardRefExoticComponent<{
     inputMode?: "none" | "text" | "tel" | "url" | "email" | "numeric" | "decimal" | "search" | undefined;
     is?: string | undefined;
     popovertarget?: string | undefined;
-    'data-size'?: "sm" | "md" | "lg" | undefined;
+    'data-size'?: "sm" | "md" | "lg" | 'xs' | undefined;
     "aria-activedescendant"?: string | undefined;
     "aria-atomic"?: (boolean | "true" | "false") | undefined;
     "aria-autocomplete"?: "none" | "inline" | "list" | "both" | undefined;
